lib/api-requestor.js

```diff
@@ -71,9 +70,6 @@ module.exports = class ApiRequestor {
       if (this._chargehound.options.protocol === 'https://') {
         req = https.request(reqOpts).setTimeout(this.getTimeout())
         connectEv = 'secureConnect'
-      } else {
```

Comment:
can we not see if the ```env``` is local or not and then use the protocol based on that?

Reply:
Done.